From cb922c943bd620a202c41e3fa16d6bd75163b8a6 Mon Sep 17 00:00:00 2001
From: Lu Yahan <yahan@iscas.ac.cn>
Date: Thu, 26 Oct 2023 10:39:53 +0800
Subject: [PATCH] [riscv] skip console-profile-wasm on unsupported platforms

Change-Id: Ifd0ab81e011a78b7722f29dbb60aa94d425a51b3
Reviewed-on: https://chromium-review.googlesource.com/c/v8/v8/+/4974897
Commit-Queue: Clemens Backes <clemensb@chromium.org>
Auto-Submit: Yahan Lu <yahan@iscas.ac.cn>
Reviewed-by: Clemens Backes <clemensb@chromium.org>
Cr-Commit-Position: refs/heads/main@{#90600}
---
 test/inspector/inspector.status | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/test/inspector/inspector.status b/test/inspector/inspector.status
index 63e8ac2746f..254f9e96bad 100644
--- a/test/inspector/inspector.status
+++ b/test/inspector/inspector.status
@@ -117,6 +117,8 @@
   # SIMD support is still in progress.
   'debugger/wasm-scope-info*': [SKIP],
   'debugger/wasm-step-after-trap': [SKIP],
+  # Generic wasm-to-js wrapper must be enabled.
+  'cpu-profiler/console-profile-wasm': [SKIP],
 
 }],  # 'arch == riscv64 or arch == riscv32 or arch == loong64'
 
-- 
2.35.1

